src/R/berekening_hulpvariabelen.R

```diff
@@ -164,23 +164,56 @@ path_to_sbp_akkervogels <- function(file = "akkervogelgebieden.shp") {
   file.path(mbag_dir, "data", "bo_vlm", file)
 }
 
+read_sbp_others <- function(
+    path,
+    soorten,
+    gebied) {
+
+  capitalize <- function(string) {
+    capped <- grep("^[A-Z]", string, invert = TRUE)
+    substr(string[capped], 1, 1) <- toupper(substr(string[capped], 1, 1))
+    return(string)
+  }
+
+  sbp_others <- st_read(path) %>%
+    st_transform(crs = 31370) %>%
+    filter(soort %in% capitalize(soorten)) %>%
```

Comment:
alternative could have been: 

`filter(tolower(soort) %in% tolower(soorten))`

Reply:
thanks that will be better